plexapi/base.py

```diff
@@ -227,7 +227,7 @@ def fetchItems(self, ekey, cls=None, container_start=None, container_size=None,
 
                     fetchItem(ekey, viewCount__gte=0)
                     fetchItem(ekey, Media__container__in=["mp4", "mkv"])
-                    fetchItem(ekey, guid__iregex=r"(imdb:\/\/|themoviedb:\/\/)")
+                    fetchItem(ekey, guid__iregex=r"(imdb:\\/\\/|themoviedb:\\/\\/)")
```

Comment:
```suggestion
                    fetchItem(ekey, guid__iregex=r"(?:imdb|themoviedb)://")
```

imho you don't need to escape `/` at all. this also optimizes regex making it more readable and skips group capture.

however, I have not tested does this actually work.

Reply:
I agree. Just remove the escape characters.